_posts/2023-05-11-Discontinuing-custom-Logstash-distribution.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title:  "Discontinuing the custom Logstash distribution"
+authors:
+- dlv
+date: 2023-05-11 12:30:00 -0500
+---
+
+One of the core [OpenSearch project principles](https://opensearch.org/about.html) is to be used everywhere. 
```

Comment:
I used the text "used everywhere" because that is exactly what I see in the project principals page: https://opensearch.org/about.html.

Perhaps this?

> One of the core [OpenSearch project principles](https://opensearch.org/about.html) includes being used everywhere: in businesses, software, and projects.

Reply:
One of the core OpenSearch Project principles is that OpenSearch can be used everywhere: in businesses, software, and projects.